sdk/textanalytics/azure-ai-textanalytics/tests/test_analyze_healthcare.py

```diff
@@ -216,7 +216,7 @@ def callback(resp):
         response = client.begin_analyze_healthcare_entities(
             docs,
             show_stats=True,
-            model_version="2022-03-01",
```

Comment:
The current list of model versions does not include 2022-03-01: https://docs.microsoft.com/en-us/azure/cognitive-services/language-service/text-analytics-for-health/language-support I wonder if the model versioning policy allows removing versions this soon.